docs/UserGuide.md

```diff
@@ -10,7 +10,7 @@ title: User Guide
     * [Viewing help: `help`](#viewing-help-help)
     * [Adding an internship application: `add`](#adding-an-internship-application-add)
     * [Listing all internship applications: `list`](#listing-all-internship-applications-list)
-    * [Listing all archived internship application: `list-archive`](#listing-all-archived-internship-applications)
+    * [Listing all archived internship application: `list-archive`](#listing-all-archived-internship-applications-list-archive)
```

Comment:
Will this make the hyperlink not linked to the header? Cause previously I purposely left this out because the command contains dash and will not be able to link to the header 😅

Reply:
Oh yeah I didn't even think of that...
Thankfully though I just tried and it is working for `remove-i` at the moment 👍